recipes/imgurpython/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: imgurpython
+  version: "1.1.7"
+
+source:
+  fn: imgurpython-1.1.7.tar.gz
+  url: https://files.pythonhosted.org/packages/d4/76/514defcb05e6d58ded427f57d736ce6dfe879e0188392385a986e8ef3b7b/imgurpython-1.1.7.tar.gz
+  md5: f2eb3a55c495d80812bae2238721da06
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests
+  run:
+    - python
+    - requests
+
+test:
+  imports:
+    - imgurpython
+    - imgurpython.helpers
+    - imgurpython.imgur
+    - imgurpython.imgur.models
+
+about:
+  home: https://github.com/Imgur/imgurpython
+  license: MIT License
```

Comment:
I guess that came from skeleton. 

I know Continuum has a set of normalized licenses that they use. Might be worth talking with them to use the same list for conda-forge (and linting against it).